composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/console": "*",
         "symfony/messenger": "^4.2 || ^5.0",
         "symfony/security-guard": "^4.0 || ^5.0",
+        "symfony/serializer": "^4.0 || ^5.0",
```

Comment:
I think this is not necessary since it is not actually used but stub files. For example, tests are run for symfony 3/4 and 5, symfony 3 tests are also passing even though it is not required here.
```suggestion
```

Reply:
If you remove `symfony/serializer`, the two new tests will not fail without `@psalm-param mixed $data`, although they do fail in the actual project. That's why I added the package. What do you think, does it make sense to keep it then? After all it's just a `dev` requirement, not a big deal...